mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/dao/DaoImplementationGenerator.java

```diff
@@ -50,6 +84,20 @@ protected String getFileName() {
   @Override
   protected JavaFile.Builder getContents() {
 
+    List<PartialClassGenerator> methodGenerators = new ArrayList<>();
```

Comment:
I will probably remove `PartialClassGenerator` soon: it's not very flexible, I prefer the pattern where the class generator passes itself to child generators and exposes higher-level hooks, such as `addEntityHelperField()` above.

It's an easy refactoring so it can be done later.